app/views/smart_answers/custom_result_full_width.erb

```diff
@@ -17,7 +17,7 @@
   <meta name="robots" content="noindex">
 <% end %>
 
-<% title = "Information based on your answers" %>
+<H1>TEMPLATE CUSTOM RESULT FULL WIDTH</h1>
```

Comment:
Is this code from debugging?

Reply:
D'oh! Yes, sorry, deleted the wrong line there. Thanks. Have fixed.

Reply:
@andysellick no worries :+1: